### PR TITLE
feat(commit.ts) : Suggestion for the user to help how to choose where to push code on remotes

### PR DESCRIPTION
**Pull request by jessicakuijer**
Here is the commit.ts file I suggest like we talked earlier.

### CODE REVIEW

**Comment by di-sukharev** [comment]
hey :) looks like your prettier just reformatted some indentation :)

**Comment by jessicakuijer** [comment]
Don't hesitate to modify it according to your indentation and style. I'm looking forward to see if it could work on your side.

**Comment by jessicakuijer** [comment]
Hi @di-sukharev ! I've added a new commit to bring much more details and correct indentation. Let me know if you can approve it.

File: src/commands/commit.ts [hunk 1]
```diff
@@ -55,9 +61,17 @@ ${chalk.grey('——————————————————')}`
     if (isPushConfirmedByUser && !isCancel(isPushConfirmedByUser)) {
       const pushSpinner = spinner();
 
-      pushSpinner.start('Running `git push`');
-      const { stdout } = await execa('git', ['push']);
-      pushSpinner.stop(`${chalk.green('✔')} successfully pushed all commits`);
+      // Get the list of remotes and ask the user to choose the desired remote
+      const remotes = await getGitRemotes();
+      const selectedRemote = await select({
```
[the remark this answers is not in the document]
Comment (jessicakuijer) [reply src/commands/commit.ts:65]: I see. I've added a new commit to the PR so if there is only one remote, the push command will automatically use that remote.

**Review by di-sukharev — APPROVED** [review]
that's a great PR! Approved. Could you please solve the conflict and im gonna merge it <3

**Comment by di-sukharev** [comment]
@jessicakuijer conflicted again after one PR was merged <3 could you solve pls and you PR is next to be merged

**Comment by jessicakuijer** [comment]
done! :)

**Pull request merged**